koji/collection-mash-split.py

```diff
@@ -405,7 +405,7 @@ def main():
         else:
             modulemd_suffix = None
 
-        mashes = [MashConfig(collection, version, "el8", "el8", "RHEL/8", modulemd_suffix=modulemd_suffix, modulemd_version=modulemd_version, modulemd_context=modulemd_context)]
+        mashes = [MashConfig(collection, version, "el8", "el8", "el8", modulemd_suffix=modulemd_suffix, modulemd_version=modulemd_version, modulemd_context=modulemd_context)]
```

Comment:
If you change this, you should probably also change EL7 to match and then our forklift config too:
https://github.com/theforeman/forklift/blob/fc38b2ecaf82cc8ee8901056308fab846fa7368a/roles/foreman_repositories/tasks/redhat_staging_repos.yml#L13
https://github.com/theforeman/forklift/blob/fc38b2ecaf82cc8ee8901056308fab846fa7368a/roles/foreman_repositories/tasks/redhat_staging_repos.yml#L24

Reply:
Yea, good call, let's just do it all.